Changelog.md

```diff
@@ -11,6 +11,15 @@ To get all details, and changes at all versions, including development ones, use
 You are reminded that Watai uses [SemVer](http://semver.org), which means upgrades that have only a patch number (last digit) change are backwards-compatible, and versions with a minor number (second digit) are API-breaking while in `0` major versions.
 
 
+v0.7 (WIP)
+----
+
+#### Minor changes
```

Comment:
This will be backward-compatible if released as 0.7, as you can see [here](https://github.com/MattiSG/Watai/blob/rename-concepts/src/controller/SuiteLoader.js#L388)  :)

We will have 1.0 remove all `Widget` and `Component` from names, so this will need a rewrite of features.